modules/040-node-manager/templates/node-group/_static_or_hybrid_script.tpl

```diff
@@ -91,7 +91,7 @@ touch /var/lib/bashible/first_run
 {{- end }}
 
 checkBashible=$(systemctl is-active bashible.timer)
-if [[ "$checkBashible" == "inactive" ]]; then
+if [[ ! "$checkBashible" == "active" ]]; then
```

Comment:
why not `[[ "$checkBashible" != "active" ]]` ?

Reply:
@yalosev, ok, fixed, they are functionally equivalent, but your suggestion looks simpler.